docs/requirements.yaml

```diff
@@ -8,11 +8,12 @@ dependencies:
     - pip
 
     - sphinxcontrib-bibtex=2.5.0
+    - mdanalysis-sphinx-theme    
```

Comment:
I forgot to merge main into the branch before editing and had independently moved danalysis-sphinx-theme from pip to conda... and then I grouped it with other sphinx things.